src/ceph-disk

```diff
@@ -2851,8 +2855,8 @@ def list_dev(dev, uuid_map, journal_map):
 
     return info
 
-def list_devices(args):
-    partmap = list_all_partitions(args.path)
+def list_devices(path=[]):
```

Comment:
OH, I just add it for convince when other function could use easily.
It looks like not make sense, I will remove the `=[]`

thanks!!
vicente